packages/gatsby/lib/schema/build-node-types.js

```diff
@@ -1,178 +1,167 @@
 // @flow
-const _ = require(`lodash`)
-const {
+import _ from 'lodash'
```

Comment:
np, didn't know whether the requires were stuff that hadn't been switch over yet. I plenty happy writing `require`